src/app/shared/components/managecitations/managecitations.component.ts

```diff
@@ -62,6 +62,9 @@ export class ManageCitationsModal implements OnInit, OnDestroy {
             this.getRegRegions(); // get list of regression regions for the region
         });
         this.modalElement = this.manageCitationsModal;
+
+        // Subscribe to server with '?bycitation=true'
+        // Copy settingservice getEntities on regions.component.ts file
```

Comment:
I think this is supposed to be for the manage citations branch (AM-89), just a note in case you start working on that branch again before this is merged into dev/back into the AM-89 branch.